doc/dashboards/jupyterlab-plugin.md

```diff
@@ -0,0 +1,65 @@
+# JupyterLab plugin
+
+```{important}
+The plugin **is not** compatible with JupyterLab `4.x`, ensure you're running `3.x`: `pip install "jupyterlab<4"`
+```
+
+You can also use the JupyterLab plugin to deploy dashboards from JupyterLab.
```

Comment:
We can link the [doc](https://github.com/ploomber/jupysql-plugin/blob/main/doc/README.md) here

Reply:
the link that you posted is developer documentation, this is meant for end-users